src/kernel/lkm.rs

```diff
@@ -115,7 +115,7 @@ impl KernelModules<'_> {
 			_ => {}
 		}
 		let modules_content = util::exec_cmd("sh", &["-c", &module_read_cmd])
-			.expect("failed to read /proc/modules");
+			.unwrap_or_else(|_| String::new());
```

Comment:
LGTM!

Maybe we can also log the error on the first run. Since pressing `r` would print it again while running the TUI.

Something like this:

```rust
		let modules_content = match util::exec_cmd("sh", &["-c", &module_read_cmd]) {
			Ok(v) => v,
			Err(e) => {
				eprintln!("{e}");
				String::new()
			}
		};
```

What do you think?

Reply:
yes, this is definitely better way to do it. I just tried the first version that comes to my mind :)